configurer/path_linux.go

```diff
@@ -3,24 +3,61 @@
 package configurer
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
+
+	"github.com/pastelnetwork/pastel-utility/constants"
 )
 
+// GetHomeDir returns the home path for Linux OS.
+func GetHomeDir() (string, error) {
```

Comment:
You've defined the same function in path_windows.go as well. 
What's the point of writing this function if you're gonna use `os.UserHomeDir()` directly.
Honestly, I don't think you should write this function as you are checking err here and the caller of this function will check err as well. So essentially you are checking same err 2 times which is redundant,

Reply:
This functions on configurer calls on runtime OS.

Reply:
So I've defined seperately.

Reply:
And this could be changed in the future. So I thought it is better to seperate this functions.